src/tensors/braidingtensor.jl

```diff
@@ -104,25 +101,12 @@ end
 # efficient copy constructor
 Base.copy(b::BraidingTensor) = b
 
-function Base.copy!(t::TensorMap, b::BraidingTensor)
-    space(t) == space(b) || throw(SectorMismatch())
-    fill!(t, zero(scalartype(t)))
-    for (f₁, f₂) in fusiontrees(t)
-        data = t[f₁, f₂]
-        if sectortype(t) == Trivial
-            r = one(scalartype(t))
-        else
-            a1, a2 = f₂.uncoupled
-            c = f₂.coupled
-            f₁.uncoupled == (a2, a1) || continue
-            braiddict = artin_braid(f₂, 1; inv=b.adjoint)
-            r = convert(scalartype(t), get(braiddict, f₁, zero(valtype(braiddict))))
-        end
-        @inbounds for i in axes(data, 1), j in axes(data, 2)
-            data[i, j, j, i] = r
-        end
+function Base.copy!(tdst::TensorMap, tsrc::BraidingTensor)
```

Comment:
This is probably equivalent to the generic `copy!(::AbstractTensorMap,::AbstractTensorMap)`